src/App.js

```diff
@@ -25,12 +26,18 @@ function App() {
                 <Route
                   exact
                   path="/"
-                  render={() => <Shopping list={data} userToken={userToken} />}
+                  render={() => <Shopping userToken={userToken} list={data} />}
                 />
                 <Route
                   path="/add"
                   render={() => <AddItem userToken={userToken} list={data} />}
                 />
+                <Route
+                  path="/detail/:itemId"
```

Comment:
Would the URL name be better if it was the items name and not the Id?

Reply:
That makes sense now. The id will always be unique and not cause any problems in the future.